packages/mobile/src/send/Send.tsx

```diff
@@ -174,46 +176,26 @@ class Send extends React.Component<Props, State> {
   }
 
   onSearchQueryChanged = (searchQuery: string) => {
-    this.props.hideAlert()
-    this.setState({
-      searchQuery,
-    })
-    CeloAnalytics.track(
-      this.props.route.params?.isRequest
-        ? CustomEventNames.request_search
-        : CustomEventNames.send_search,
-      {
-        query: searchQuery,
-      }
-    )
     this.throttledSearch(searchQuery)
   }
 
   onSelectRecipient = (recipient: Recipient) => {
     this.props.hideAlert()
     const isRequest = this.props.route.params?.isRequest ?? false
-    CeloAnalytics.track(CustomEventNames.send_input, {
-      selectedRecipientAddress: recipient.address,
-    })
 
     if (!recipient.e164PhoneNumber && !recipient.address) {
       this.props.showError(ErrorMessages.CANT_SELECT_INVALID_PHONE)
       return
     }
 
-    if (
-      (recipient.e164PhoneNumber && recipient.e164PhoneNumber !== defaultRecipientPhoneNumber) ||
-      (recipient.address && recipient.address !== defaultRecipientAddress)
-    ) {
-      this.props.storeLatestInRecents(recipient)
-    }
+    this.props.storeLatestInRecents(recipient)
 
     CeloAnalytics.track(
-      this.props.route.params?.isRequest
+      isRequest
         ? CustomEventNames.request_select_recipient
         : CustomEventNames.send_select_recipient,
       {
-        query: this.state.searchQuery,
+        recipientKind: recipient.kind,
```

Comment:
I think we should also collect the `query`. It will be informative to know if people are selecting recipients via scrolling vs. searching

Reply:
The query can have names in it, which is PII

Reply:
Good point. How about we just pass a boolean depending on if `query.length > 0`? What specifically is entered is less important IMO anyways